### ISSUE TITLE
Ruby 2.6, warning: BigDecimal.new is deprecated

### ISSUE THREAD

**Issue by thibaudgg** [Barzahlen/camt_parser#16]
Hi, got this warning message on Ruby 2.6

```
~/.gem/ruby/2.6.0/gems/camt_parser-2.3.5/lib/camt_parser/misc.rb:9: warning: BigDecimal.new is deprecated; use BigDecimal() method instead.
```

Thanks for your gem!

**Comment by tobischo** [comment]
I just released v2.3.6 where this is addressed

**Comment by thibaudgg** [comment]
@tobischo awesome, thank you! 💌
